ui/server/src/main/scala/pl/touk/nussknacker/ui/definition/DefinitionPreparer.scala

```diff
@@ -51,28 +51,28 @@ object DefinitionPreparer {
     => objectDefinition.hasNoReturn).tupled
 
     //TODO: make it possible to configure other defaults here.
-    val base = NodeGroup("base", List(
+    val base = NodeGroup(ComponentGroupName("base"), List(
```

Comment:
It looks weird that **NODE**Group has **COMPONENT**GroupName. Maybe also change `NodeGroup` -> `ComponentGroup` and `NodeToAdd` -> `ComponentTemplate`?